modules/swagger-core/src/main/java/io/swagger/v3/core/util/ParameterProcessor.java

```diff
@@ -167,11 +165,7 @@ public static Parameter applyAnnotations(
                 } catch (Exception e) {
                     LOGGER.error("failed on " + annotation.annotationType().getName(), e);
                 }
-            } else if (
-                        annotation.annotationType().getName().equals("javax.validation.constraints.NotNull") ||
-                        annotation.annotationType().getName().equals("javax.validation.constraints.NotBlank") ||
-                        annotation.annotationType().getName().equals("javax.validation.constraints.NotEmpty")
-                    ) {
+            } else if (NOT_NULL_ANNOTATIONS.contains(annotation.annotationType().getSimpleName())) {
```

Comment:
would use ModelResolver.NOT_NULL_ANNOTATIONS as single source to avoid duplication